### PR TITLE
T8710: entityDataCachePaths configurations

### PR DESCRIPTION
**Pull request by ugochimobi**
This might be wrong; I mean, I might have not done it properly, please look and correct appropriately

### CODE REVIEW

**Review by RhinosF1 — CHANGES_REQUESTED** [review]
Syntax

**Comment by ugochimobi** [comment]
tbh, I don't know what the correct syntax is, please if you can, check the syntax yourself

File: Wikibase.php [hunk 1]
```diff
@@ -108,6 +108,20 @@
 		// hardcoded internal handling
 		'html',
 	];
+	$wgWBRepoSettings['entityDataCachePaths'] = function() [
```
Comment (RhinosF1) [inline Wikibase.php:111]: Should be a { instead of [ at the end. Same to close

Comment (ugochimobi) [reply Wikibase.php:111]: I so much thought of that, I was just starring at it lol

**Review by RhinosF1 — APPROVED** [review]

File: Wikibase.php [hunk 2]
```diff
@@ -108,6 +108,20 @@
...
+				str_replace( '$1', 'Special:EntityData/{entity_id}.json', $GLOBALS['wgArticlePath'] ),
+				'revision={revision_id}'
+			),
+			// Turtle from Query Service updater
+			wfAppendQuery(
+				str_replace( '$1', 'Special:EntityData/{entity_id}.ttl', $GLOBALS['wgArticlePath'] ),
+				'flavor=dump&revision={revision_id}'
+			),
+		];
+	};
```
Comment (RhinosF1) [inline Wikibase.php:124]: Not sure the ; is needed

Comment (ugochimobi) [reply Wikibase.php:124]: okay

Comment (ugochimobi) [reply Wikibase.php:124]: welp, as it stands, I think it is needed

File: Wikibase.php [hunk 3]
```diff
@@ -108,6 +108,14 @@
...
 		'html',
 	];
+	$wgWBRepoSettings['entityDataCachePaths'] = [
+		// // JSON from entity page JS, compare wikibase.entityPage.entityLoaded.js
+		'/wiki/Special:EntityData/{entity_id}.json?revision={revision_id}',
+		// Turtle from Query Service updater, compare WikibaseRepository.java
+		'/wiki/Special:EntityData/{entity_id}.ttl?flavor=dump&revision={revision_id}',
+		// third pattern with high volume of requests in Hive, source unknown
+		'/wiki/Special:EntityData?id={entity_id}&revision={revision_id}&format=json',
+	];
```
Comment (ugochimobi) [inline Wikibase.php:118]: This is how simple it should look, instead of the former.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
This is equivalent to the default, and unnecessary from what I can see.

**Comment by ugochimobi** [comment]
@Universal-Omega you should probably see this https://phabricator.miraheze.org/T8710

**Review by Universal-Omega — COMMENTED** [review]
This particular config value is just copied and pasted from Wikimedia config. Can you please say how it'll fix the mentioned task, and why it's necessary for us?

**Comment by Universal-Omega** [comment]
This wouldn't fix the issue, as its basically the same as the default, plus the mentioned task is already resolved.

**Pull request closed without merging**